README.md

```diff
@@ -50,6 +51,9 @@ module "scc_wp" {
   region            = "us-south"
   resource_group_id = "65xxxxxxxxxxxxxxxa3fd"
   resource_key_tags = ["scc-wp-tag"]
+  parameters = {
+    cloud_monitoring_connected_instance : "crn:v1:bluemix:public:sysdig-monitor:us-south:a/xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX:xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX::"
+  }
```

Comment:
thats not the usage for the module. It should be using the new variable you added:
```
cloud_monitoring_instance_crn = crn:v1:bluemix:public:sysdig-monitor:us-south:a/xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX:xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX::
```